src/components/common/ColonyHome/ColonyFundingWidget/ColonyFundingWidget.tsx

```diff
@@ -63,29 +91,27 @@ const ColonyFundingWidget = (/* { currentDomainId }: Props */) => {
           <FormattedMessage {...MSG.title} />
         </NavLink>
       </Heading>
-      {/* {data && !isLoadingTokenBalances ? ( */}
       <ul data-test="availableFunds">
         {tokens?.map((token) => (
-          <li key={token.tokenAddress}>
+          <li key={token?.tokenAddress}>
```

Comment:
This `li` could be given a name and moved into own component (`AvailableFundItem`)?

Reply:
Good suggestion, not doing it: https://github.com/JoinColony/colonyCDapp/pull/74#discussion_r1131145391